### DISCUSSION TITLE
loggroup queue is full

### DISCUSSION THREAD

**Discussion by onejimmyboy** [alibaba/ilogtail#1421] [Help]
version:1.8.4
配置文件已经被删除很久了，查看logtail_plugin.LOG日志，依然还是在打印这个警告日志。
![image](https://github.com/alibaba/ilogtail/assets/57432151/debc8c6c-9f87-471d-8a82-d294e69c16e6)

![image](https://github.com/alibaba/ilogtail/assets/57432151/c503cce1-7e6c-4698-936e-97a04538b509)

**Comment by Takuka0311** [comment]
看第二个截图，total 52个文件，有列全了看看吗？
另外看目录是通过ConfigServer远程配置的采集配置，如果ConfigServer处这个配置没有删除，logtail还是会一直把它拉下来的。
可能的解决方法：
1. 删除ConfigServer处的这个配置
2. 这个报错一般是因为日志量比较大，可以调大logtail资源限制试试

**Reply by onejimmyboy** [reply]
回答第一个问题：total52，列全了，实际在remote_yaml_config.d目录下只有13个配置文件
第二个问题：configserver中配置确实有被删除，如果没有删除，remote_yaml_config.d目录下这个配置文件不会从有到无的
第三个问题：ilogtail资源我们设置的比较大。cpu_usage_limit和mem_usage_limit同时这是的是2000
